src/main/java/io/codearte/jfairy/producer/person/locale/ContinentalAddress.java

```diff
@@ -0,0 +1,29 @@
+package io.codearte.jfairy.producer.person.locale;
+
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+
+/**
+ * An address format typical for European countries but the UK and ex-Soviet union.
+ */
+public abstract class ContinentalAddress extends AbstractAddress {
```

Comment:
I'm not sure if 2 levels of abstractions are needed here. Maybe it would be better to leave the implementation of the methods that most countries use in `AbstractAddress`, and just override it in `Ge`, `En` and `Zh`?

Reply:
I believe that the order of address parts ( street > number, ZIP > city) is really quite common for European countries, but nowhere else.
What is wrong with one intermediate class shared by four implementations?